rubocop/lib/rubocop.yml

```diff
@@ -0,0 +1,335 @@
+require: rubocop-rspec
+
+AllCops:
+  Exclude:
+    - bin/**/*
+
+# LINT
+
+Lint/AmbiguousBlockAssociation:
+  # `scope :name, -> (param) { ... }` is a valid syntax
+  Enabled: false
+
+Lint/AmbiguousOperator:
+  # REVIEW
+  Enabled: true
+
+Lint/AssignmentInCondition:
+  # REVIEW
+  # It allows to use `if (foo = 123)` instead of `if foo = 123`
+  Enabled: true
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
+
+Lint/HandleExceptions:
+  Enabled: false
+
+Lint/LiteralInCondition:
+  # backtics? seriously?
+  Enabled: false
```

Comment:
Why? And also, in my opinion, `.js.rb` files should not be ignored in this gem. The project that uses the gem is responsible for this.